src/praisonai-agents/praisonaiagents/bus/bus.py

```
@@ -338,6 +354,54 @@ def clear_subscribers(self):
+        return bool(self._sinks)
+    
+    def _dispatch_to_sinks(self, event: Event) -> None:
+        """Best-effort durable persistence: never raises into the caller."""
+        if not self._sinks:
+            return
+        with self._lock:
+            sinks = list(self._sinks)
+        for sink in sinks:
+            try:
+                sink.append(event)
+            except Exception as e:  # pragma: no cover - defensive
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)